docs/source/integrations/mobile/tvos-fairplay-sdk.md

```diff
@@ -0,0 +1,124 @@
+
+Current release: v0.0.1 (10)
+
+- Overview
+- Requirements
+- Xcode integration
+- Information about Application Transport Security (ATS)
+- Example framework usage
+- Limitations
+- Known Issues
+- Release Notes
+
+## Overview
+
+VUDRMFairPlay SDK enables Apple’s AVPlayer from AVFoundation to securely request licenses from Vualto’s VUDRM cloud based DRM platform using an extended instance of AssetResourceLoaderDelegate.
+
+Make use of Apple’s AVPlayer and AVPlayerViewController to present users with the platform default skins, with DRM content. Or create your own video player based on AVPlayer.
+
+As an evolution of the the hugely successful VUPLAY SDK, this SDK has a number of key benefits:
+
+ - Complete control over the entire AVPlayer lifecycle
+ - Numerous optimisations to enable faster playback
+ - Bitcode support
+ - Support for building/linking against the simulators
+
+The SDK is fully supported in both Objective-C and Swift applications. 
+
+A demo application written in Swift is available on request. Please contact [support@vualto.com](support@vualto.com) to request a download of the SDK.
+
+## Requirements
+
+- Minimum tvOS deployment target of 11.x or higher
+- Xcode 10.2
+- Swift 5
+
+## Xcode Integration
+
+1. Copy the vudrmFairPlay.framework into your application's project directory
+2. Navigate to your target and add vudrmFairPlay.framework to your ‘Embedded Binaries’. You should see that vudrmFairPlay.framework also now appears under ‘Linked Frameworks and Libraries’:
+
+## Information about application transport security (ATS)
+
+tvOS 9 introduces Application Transport Security (ATS) which restricts the use of insecure HTTP.
+In order to permit playback of content over insecure HTTP exemptions need to be added into your application's Info.plist. For example to disable ATS for any connection you would add the following into your application's Info.plist:
+
+```xml
+<dict>
+    <key>NSAppTransportSecurity</key>
+    <dict>
+        <key>NSAllowsArbitraryLoads</key> <true/>
+    </dict>
+</dict>
+```
+
+More information about ATS can be found in Apple’s TechNote:
+[https://developer.apple.com/library/prerelease/ios/technotes/App-Transport-Security-Technote/](https://developer.apple.com/library/prerelease/ios/technotes/App-Transport-Security-Technote/)
+
+## Preparation
+
+Before you can use the demo project or framework in your own project, for each instance you will require a URL to correctly prepared content and a VUDRM token which must be correctly generated for the type of instance you wish to create. An example VUDRM token type template available for use with tvOS is:
+
+- Fairplay Rental `{"type": "r","duration_rental": 3600}`
+
+For further information about VUDRM please contact us, or refer to our documentation:
+[https://docs.vualto.com/en/latest/vudrm.html](https://docs.vualto.com/en/latest/vudrm.html)
```

Comment:
Should you link directly to the VUDRM token docs here?
https://docs.vualto.com/projects/vudrm/en/latest/VUDRM-token.html#drm-policy

Reply:
You suggested I refer to the VUDRM token docs directly in the iOS version of this document.

Whilst the tvOS version does not offer persist / offline functionality, I believe that the VUDRM token docs are still relevant to the clients token.

Reply:
@macmagician Sorry I don't quite follow. This section on the tvos docs is talking solely about the VUDRM token. It seems sensible to me that it links directly to the VUDRM token docs.

Reply:
Ahh I see you mean here:
https://docs.vualto.com/projects/vudrm/en/latest/integrations/mobile/ios-fairplay-sdk.html#preparation
has a link directly to the VUDRM docs root.

In that case I think that for both docs it should probably read:
> For further information about VUDRM tokens please contact us, or refer to our documentation: https://docs.vualto.com/projects/vudrm/en/latest/VUDRM-token.html

Reply:
Yes, agreed, let's use 
For further information about VUDRM tokens please contact us, or refer to our documentation: https://docs.vualto.com/projects/vudrm/en/latest/VUDRM-token.html